backend/src/prompts/templates/answer-user-ques.j2

```diff
@@ -9,23 +9,24 @@ User's question is:
 Once you generate an answer:
 - **Check** if the answer completely and accurately addresses the user's question.
 - **Determine** if the answer is valid, based on the content provided.
+- **Provide** a reason for the validity or invalidity of the answer.
+- **Indicate** if a more general web search is required then indicate that.
 
 Reply only in JSON format with the following structure:
 
 ```json
 {
     "answer": "The answer to the user's question, based on the content provided",
-    "is_valid": true or false,
+    "search_more": true or false,
     "validation_reason": "A sentence explaining whether the answer is valid or not, and why"
 }
 
 
-
 ### **Explanation:**
 
 1. **Answer**: The LLM generates an answer based on the user’s question and the provided content.
-2. **Validity Check**: The LLM checks if its generated answer is complete and correct. This could be based on factual accuracy, coverage of the query, or relevance to the user's question.
-3. **Validation Reason**: The LLM explains why the answer is valid or invalid.
+2. **Search More**: If a more general web search using a web search engine is required then indicate that, it should set this field to true.
+4. **Validation Reason**: The LLM explains why the answer is valid or invalid.
```

Comment:
Should this now be named something like "search more reason" and should the text be changed to address the changes on line 28

Reply:
Changed it to search_more_reason